reviewdog.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/haya14busa/reviewdog/diff"
 )
 
+var NoComment = 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var NoComment should have comment or be unexported